04-Multitasking/context_switch.S

```diff
@@ -1,5 +1,16 @@
 .syntax unified
 
+/* check the situaion and determine the path to return */
+.macro ret_path, scratch
+	mov \scratch, #0xfffffff0
+	cmp lr, \scratch  /* lr - 0xfffffff0 */
+
+	ittt ls /* `ls` condition means unsinged lower or the same */
```

Comment:
@jserv , The original coding style in `context_switch.S` use C-style comment. Should I modify all of them?